src/Utility.js

```diff
@@ -1,14 +1,10 @@
-/** @typedef {import('./Middleware.js').ArrayMapCallback} ArrayMapCallback */
-
 /**
- * @param {number}
```

Comment:
old definitions were not picked up by the IDE at all - I think those can only be used for actual function declarations and not for defining types when assigning to a variable like this